provisioning/miniprov/hare_mp/main.py

```diff
@@ -657,11 +674,11 @@ def main():
                                nargs='?',
                                help='Target directory; defaults to /tmp/hare.')
 
-    add_subcommand(subparser,
-                   'reset',
-                   help_str='Resets temporary Hare data and configuration',
-                   handler_fn=reset,
-                   config_required=False)
+    add_file_argument(
```

Comment:
Why this change is needed?

Reply:
In the reset stage, we are calling the `init` stage which requires the `--file` argument. For this, we have added these changes here.

Reply:
Addressed.